plugins/woocommerce-admin/client/products/fields/options/options.tsx

```diff
@@ -40,9 +42,29 @@ export const Options: React.FC< OptionsProps > = ( {
 
 	return (
 		<AttributeControl
-			attributeType="for-variations"
 			value={ attributes }
 			onChange={ handleChange }
+			onModalClose={ ( attribute ) => {
+				if ( attribute ) {
+					return;
+				}
+				recordEvent( 'product_add_options_modal_cancel_button_click' );
```

Comment:
This event should only be recorded when the modal is actually canceled. It currently gets logged every time the modal is closed, either by "Cancel" or "Add"

Reply:
Good catch!  Updated the callback handlers a bit in https://github.com/woocommerce/woocommerce/pull/36563/commits/b8e59db9915da402182b0115ef58bcccb613a3cc to be more explicit.  It's a bit more verbose but much easier for consumers to identify the events.